tests/system/python/test_e2e_csv_PI.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test system/python/test_e2e_csv_PI.py
+
+"""
+import os
+import http.client
+import json
+import time
+import shutil
+import pytest
+from collections import Counter
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2018 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+CSV_NAME = "sample.csv"
+CSV_HEADERS = "ivalue,fvalue,svalue"
+CSV_DATA = [{'ivalue': 1, 'fvalue': 1.1, 'svalue': 'abc'},
```

Comment:
This should also be possible via a given CSV; We can doc the template and if provided use that instead of creating value on the fly